docs/README.md

```diff
@@ -15,12 +15,10 @@ Forge is a free, open-source modding API all of your favourite mods use!
 * [Discord]
 * [Documentation]
 
-# Installing NeoForged
-
-Go to [CurseForge][CurseForge] project, select the minecraft version and installer, and run it.
+# Installing NeoForge
 
 You can download the installer for the *Recommended Build* or the
- *Latest build* there. Latest builds may have newer features but may be
+ *Latest build* at [NeoForged]. Latest builds may have newer features but may be
  more unstable as a result. The installer will attempt to install Forge
```

Comment:
Missed Forge -> NeoForge change

Reply:
This appears correct to me in the latest commit already